lib/src/remote/base_remote_appender.dart

```diff
@@ -170,19 +170,27 @@ class SimpleJobDef {
   SimpleJobDef({required this.runner});
 
   final SimpleJobRunner runner;
+
+  int errorCount = 0;
+  DateTime? lastError;
 }
 
 class SimpleJobQueue {
-  SimpleJobQueue({this.maxQueueSize = 100});
+  SimpleJobQueue({
+    this.baseWaitSeconds = 10,
+    this.maxRetries = 5,
+    Future<void> Function(Duration)? delayFunction,
+  }) : _delayFunction = delayFunction ?? Future.delayed;
```

Comment:
Is this the public interface which can be used to configure things from the app?

Reply:
No, this is internal. Do you think it's worth exposing this?

Reply:
How does the app configure the details? Or doesn't it?

I'd go to 3 max retries personally

Reply:
It doesn't, but could probably prop drill it?

Ill change default to 3